src/homepage.js

```diff
@@ -0,0 +1,125 @@
+  numberOfCards.innerHTML = ` [${howmany}]`;
+};
+
+const createPokeCard = (pokemon) => {
+  const cardContainer = document.createElement('div');
+  const card = document.createElement('div');
+  const imgContainer = document.createElement('img');
+  const cardBody = document.createElement('div');
+  const cardTitle = document.createElement('h5');
+  const likeIconContainer = document.createElement('a');
+  const likeIcon = document.createElement('i');
+  const likeCount = document.createElement('span');
+  const cardText = document.createElement('p');
+  const commentButton = document.createElement('a');
+  cardContainer.classList.add('col-12', 'col-sm-6', 'col-md-3', 'pt-4');
+  card.classList.add('card', 'border', 'border-2', 'border-black', 'card-style');
+  imgContainer.classList.add('card-img-top');
+  imgContainer.alt = `Pokemon Card image of ${pokemon.name}`;
+  imgContainer.src = pokemon.images.small;
+  cardBody.classList.add('card-body', 'row', 'justify-content-center', 'align-items-center');
+  cardTitle.classList.add('card-title', 'pb-2', 'col-6', 'd-flex', 'justify-content-center', 'align-items-center');
+  cardTitle.innerText = pokemon.name;
+  likeIconContainer.classList.add('col-4', 'pb-2', 'd-flex', 'justify-content-center', 'align-items-center', 'iconLike');
+  likeIcon.classList.add('fas', 'fa-heart', 'fa-2x', 'col-4', 'd-flex', 'justify-content-center', 'align-items-center');
+  likeCount.classList.add('col-2', 'd-flex', 'justify-content-center', 'align-items-center');
+  likeCount.id = `likeCount${pokemon.id}`;
+  likeCount.innerText = 0;
+  cardText.classList.add('card-text', 'd-flex', 'justify-content-evenly', 'align-items-center');
+  cardText.innerText = pokemon.flavorText || 'No Description';
+  commentButton.classList.add('btn', 'btn-primary', 'col-6');
+  commentButton.id = `commentButton_${pokemon.id}`;
+  commentButton.innerText = 'Comments';
+
+  likeIconContainer.appendChild(likeIcon);
+  cardBody.appendChild(cardTitle);
+  cardBody.appendChild(likeIconContainer);
+  cardBody.appendChild(likeCount);
+  cardBody.appendChild(cardText);
+  cardBody.appendChild(commentButton);
+  card.appendChild(imgContainer);
+  card.appendChild(cardBody);
+  cardContainer.appendChild(card);
+  PokeCards.appendChild(cardContainer);
+  /* cardContainer.addEventListener('click', () => {
+          const modal = document.getElementById('modal');
+          const modalImg = document.getElementById('modal-img');
+          const modalTitle = document.getElementById('modal-title');
+          const modalText = document.getElementById('modal-text');
+          const modalClose = document.getElementById('modal-close');
+          modal.style.display = 'block';
+          modalImg.src = pokemon.images.large;
+          modalTitle.innerText = pokemon.name;
+          modalText.innerText = pokemon.text;
+          modalClose.addEventListener('click', () => {
+              modal.style.display = 'none';
+          })
+      } */
```

Comment:
- [ ] You should keep your codebase professional. Please remove commented-out code blocks. You can use git history to see removed code blocks

Reply:
Thank you for your feedback @sinansevgi. The amends are made it in commit e7cead6.